web/html/src/manager/admin/payg-shared/sshData/payg-ssh-data-view.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+
+import { paygProperties } from "manager/admin/payg/payg";
+
+type Props = {
+  payg: paygProperties;
+  isInstance: boolean;
+};
+
+const PaygShhDataView = (props: Props) => {
+  return (
+    <div>
+      <React.Fragment>
```

Comment:
Both of these fragments are redundant. Fragments are required when you want to return multiple children from a single component as in a partial, but when you have wrapping divs this is not needed. Also, the wrapping divs seem to be duplicated here.